webapp/src/main/webapp/iaf/frankdoc/views/attribute-description.html

```diff
@@ -0,0 +1,20 @@
+<div>
+  <div ng-if="! attr.enum">{{ attr.description }}</div>
+  <div ng-if="attr.enum">
+    {{ attr.description }}
+    <p>
+    <p>
+    <table class="attributeEnumValues">
+      <thead>
+        <tr>
+          <th>Value</th>
+          <th>Meaning</th>
+        </tr>
+      </thead>
+      <tr ng-repeat="enumValue in enumValuesOfAttribute(attr)">
```

Comment:
Hier zou je (om het eerdergenoemde performance probleem op te lossen) de two way binding uit kunnen zetten. De content van de functie veranderd immers niet. Die doe je door twee dubbele-punten achter elkaar te schrijven voor de functie aanroep `::enums[attr.enum]`

Reply:
Hier los ik het performance probleem op door :: toe te voegen voor de call naar enumValuesOfAttribute. Is dat wat je bedoelt?